community/OM-Operator-Management/cnv-samples/virtual-machines/fedora-rootdisk-and-datadisk.yaml

```diff
@@ -0,0 +1,229 @@
+                  instancetype:
+                    kind: virtualmachineclusterinstancetype
+                    name: u1.small
+                  preference:
+                    kind: virtualmachineclusterpreference
+                    name: fedora
+                  runStrategy: Halted
+                  template:
+                    metadata:
+                      creationTimestamp: null
+                      labels:
+                        network.kubevirt.io/headlessService: headless
+                    spec:
+                      architecture: amd64
+                      domain:
+                        devices:
+                          autoattachPodInterface: false
+                          disks:
+                            - disk:
+                                bus: virtio
+                              name: disk-vm02
+                          interfaces:
+                            - masquerade: {}
+                              name: default
+                        machine:
+                          type: pc-q35-rhel9.4.0
+                        resources: {}
+                      networks:
+                        - name: default
+                          pod: {}
+                      subdomain: headless
+                      volumes:
+                        - dataVolume:
+                            name: fedora-vm02
+                          name: rootdisk
+                        - cloudInitNoCloud:
+                            userData: |
+                              #cloud-config
+                              chpasswd:
+                                expire: false
+                              password: change-me-5738
+                              user: fedora
+                              #ssh_authorized_keys:
+                              #  - ssh-rsa ssh-rsa...
+                          name: cloudinitdisk
+                        - dataVolume:
+                            name: dv-fedora-vm02
+                          name: disk-vm02
+
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: create-vm-with-rootdisk-only
+        spec:
+          namespaceSelector:
+            exclude:
+              - kube-*
+            include:
+              - default
+          remediationAction: inform
+          severity: low  # Present for console wizard compatibility
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: policy.open-cluster-management.io/v1
+                kind: ConfigurationPolicy
+                metadata:
+                  name: create-vm-with-rootdisk-only
+                spec:
+                  namespaceSelector:
+                    exclude:
+                      - kube-*
+                    include:
+                      - default
+                  object-templates:
```

Comment:
I think this extra ConfigurationPolicy layer was an accident?

Reply:
I decided to drop this example.